editor/project_manager.cpp

```diff
@@ -463,6 +464,10 @@ void ProjectManager::_run_project_confirm() {
 		args.push_back("--path");
 		args.push_back(path);
 
+		if (open_in_verbose_mode) {
```

Comment:
How to run in verbose mode? The option is available only for editing.

Reply:
Ah, good point - that is an artifact from when holding down the Alt key while clicking "Run" would let it run in verbose mode. I'll remove that for now, since there's no way to reach it